parser-worker/webpack.config.js

```diff
@@ -9,12 +9,17 @@ const createBaseConfig = (env) => {
     mode: development ? "development" : "production",
     target: "webworker",
     entry: {
-      "reveal.parser.worker": "./reveal.parser.worker.ts",
+      "reveal.parser.worker": "./index.ts",
     },
     output: {
-      filename: "[name].js",
-      sourceMapFilename: "[name].map",
-      libraryTarget: "umd",
+      filename: '[name].js',
+      path: path.resolve(__dirname, 'dist'),
```

Comment:
Shouldn't be here, it's overridden in configs below. 

```suggestion
```

https://github.com/cognitedata/reveal/pull/1316/files#diff-95be1f30cccebb8f33c2197f0b594800fe75740e9a7510dfab07d84a455cb435R51

https://github.com/cognitedata/reveal/pull/1316/files#diff-95be1f30cccebb8f33c2197f0b594800fe75740e9a7510dfab07d84a455cb435R39